appveyor.yml

```diff
@@ -0,0 +1,160 @@
+version: '{branch}-{build}'
+os: Visual Studio 2015
+clone_folder: C:\projects\oiio
+test: off
+configuration:
+    - Release
+    # - Debug
+platform:
+    - x64
+# branches:
+#   only:
+#     - master
+environment:
+  matrix:
+    # - CMAKE_PLATFORM: "Visual Studio 14 2015"
+    #   PYTHON_DIR: "C:\\Python27"
+    - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
+      PYTHON_DIR: "C:\\Python27-x64"
+      tbs_arch: "x64"
+      tbs_tools: "msvc14"
+      tbs_static_runtime: 0
+      # BUILD_STATIC: 1
+
+install:
+  - cinstall: python
+  - ps: if($env:build_platform -eq 'x64') {
+          $env:vcvar_arg = 'x86_amd64';
+        }
+        else {
+          $env:vcvar_arg = 'x86';
+        }
+  - call "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall.bat"
```

Comment:
I'm not quite sure either. A lot of this whole process is black magic.

I'll comment these lines out and try one more build, just to see if it works. But if that breaks things, I'm going to commit what I have for now rather than try to understand why it didn't work.

I need to make a second pass over this all (including making it run the testsuite in addition to performing the build), but I'm really itching to commit what I have so far. I've been working on this in tiny bits of free time here and there for weeks.